app/js/UserPic.js

```diff
@@ -73,90 +75,116 @@ var pickPhoto = function(callback) {
     });
 };
 
-var UserPic = React.createClass({
-    getUserInfo: function(callback) {
-        forceClient.sendRequest('/services/data', '/v36.0/chatter/users/me', 
-                                function(response) {
-                                    console.log(JSON.stringify(response));
-                                    callback(response);
-                                },
-                                function(error) {
-                                    console.log('Failed to get user info:' + error);
-                                }, 
-                                'GET', 
-                                {}, 
-                                {'X-Connect-Bearer-Urls': 'true'});
+module.exports = React.createClass({
+
+    getInitialState() {
+        return {
+          refreshing: false
+        };
+    },
+
+    onRefresh() {
+        this.refreshUserInfo();
+    },
 
+    getUserInfo(callback) {
+        forceClient.sendRequest('/services/data', '/v36.0/chatter/users/me', 
```

Comment:
Not a big deal but, instead of changing the version number of the API in multiple places, you could have a constant with the version number. This way you don't have to change it in multiple places when we want to update the version number of the API. :)